Makefile

```
@@ -33,6 +30,22 @@ frontend:
 	docker build --build-arg CONFIGURATION=local -t t4c/client/frontend -t $(LOCAL_REGISTRY_NAME):$(REGISTRY_PORT)/capella/collab/frontend frontend
 	docker push $(LOCAL_REGISTRY_NAME):$(REGISTRY_PORT)/capella/collab/frontend
 
+capella:
+	for version in $(CAPELLA_VERSIONS)
+	do $(MAKE) capella-$$version
+	done
+
+capella-%: registry
+	$(CAPELLA_DOCKERIMAGES) CAPELLA_VERSION=$* capella/remote capella/readonly
```

```suggestion
capella:
	for version in $(CAPELLA_VERSIONS)
	do $(CAPELLA_DOCKERIMAGES) CAPELLA_VERSION=$$version capella/remote capella/readonly
	done
```

Makes it clearer and keeps it simpler. We shouldn't start building commands around the Capella Docker images targets, they belong to the Capella Docker images Makefile.